src/Community/Block/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Action, Content, Icon, Header, Wrapper } from './styles'
+
+export default function Block({ action, children, icon, large, title }) {
+  const hasAction = !!action
```

Comment:
Random q for @iAdramelk: Do you usually prefer to use function components in React/Next (instead of classes)? Apparently they officially prefer them... See https://reactjs.org/docs/hooks-intro.html

Reply:
Yep, functional components are a lot easier to read for me, and hooks make using state and side effects a lot easier and cleaner.

Reply:
Thanks. Will keep this in mind.